### PR TITLE
Nightly 11 10 20

### PR DESCRIPTION
**Pull request by Eutherin**
removed a few duplicate debug falses, removed reference to /vote in the resource pack loader

### CODE REVIEW

**Comment by Eutherin** [comment]
https://github.com/Adriftus-Studios/network-script-data/issues/217

**Comment by Eutherin** [comment]
https://github.com/Adriftus-Studios/network-script-data/issues/197

**Comment by Eutherin** [comment]
https://github.com/Adriftus-Studios/network-script-data/issues/129

**Comment by Eutherin** [comment]
https://github.com/Adriftus-Studios/network-script-data/issues/98 @Ph4i1ur3

File: denizen_scripts/survival/repo_link/systems/server_restarter.dsc [hunk 1]
```diff
@@ -19,7 +19,7 @@ survival_restart_timer:
       - wait 4.5m
       - announce <&b> "The server will be performing a scheduled restart in 30 seconds. Please reach a safe location."
       - repeat 29:
-        - actionbar "<&c>The server will be restarting in <element[31].-[<[value]>]> seconds."
+        - actionbar "<&e>The server will be restarting in <&c><element[31].-[value]> <&e>seconds."
```
Comment (Ph4i1ur3) [inline denizen_scripts/survival/repo_link/systems/server_restarter.dsc:22]: This is invalid.

File: denizen_scripts/survival/repo_link/systems/vote_handlers.dsc [hunk 2]
```diff
@@ -11,28 +11,36 @@ vote_receiver:
...
-        - narrate "<&e>Your inventory was full and <&a>a <item[daily_vote_key].display>s<&e> was deposited into your enderchest!"
-      - if <player.flag[inventory_daily_key_offline]> > 0:
-        - if <player.flag[inventory_daily_key_offline]> > 1:
-        - narrate "<&e>You received <player.flag[inventory_daily_key]> <item[daily_vote_key].display>s<&e> while offline!"
-      - else:
-        - narrate "<&e>You received <&a>a <item[daily_vote_key].display><&e> while you were offline!"
-      - if <player.flag[daily_key_pending]> > 0:
-        - if <player.flag[daily_key_pending]> > 1:
-          - narrate "<&c>You have <player.flag[daily_key_pending]> <item[daily_vote_key].display>s<&c> waiting for you at <&3>/<&b>recoverkeys."
+      - if <player.has_flag[nderchest_daily_key_offline]>:
```
Comment (Ph4i1ur3) [inline denizen_scripts/survival/repo_link/systems/vote_handlers.dsc:16]: Typo, missing the e on enderchest

Comment (Eutherin) [reply denizen_scripts/survival/repo_link/systems/vote_handlers.dsc]: Fixed.

File: denizen_scripts/survival/repo_link/systems/server_restarter.dsc (hunk 1 above)
Comment (Eutherin) [reply denizen_scripts/survival/repo_link/systems/server_restarter.dsc:22]: image

Comment (Eutherin) [reply denizen_scripts/survival/repo_link/systems/server_restarter.dsc:22]: https://github.com/Adriftus-Studios/network-script-data/issues/217

**Review by BehrRiley — APPROVED** [review]

**Pull request merged**